templates/snippets/editoria11y.html

```diff
@@ -0,0 +1,13 @@
+{% load static %}
+{% if perms.wagtailadmin.access_admin %} {# use same check as wagtailuserbar #}
+{# jquery included in base template #}
+<link rel="stylesheet" media="screen" href="{% static "vendor/editoria11y-1.0.6/css/editoria11y.css" %}">
+<script src="{% static "vendor/editoria11y-1.0.6/js/editoria11y-prefs.js" %}"></script>
+<script src="{% static "vendor/editoria11y-1.0.6/js/editoria11y-localization.js" %}"%></script>
+{% if request.is_preview %}
+<script>
+ed11yAlertMode = "assertive";  {# not sure this is actually working #}
+</script>
+{% endif %}
```

Comment:
you could put this logic inside of `editoria11y-prefs.js` and then `{% include %}` it instead of using `{% static %}` so that the logic would work inside the JS file. just an idea

Reply:
But then wouldn't we have to the prefs js a template file instead? Can we do that with js? (I'm trying to remember if we have... would it just all be inline, then?)

Reply:
Idk about the file extension, but I've definitely done it at least once before. This approach doesn't work if we always want the latest version though, because we have to keep a static copy to use as a template, so probably not worth it